docs/api.rst

```diff
@@ -1,84 +1,95 @@
-.. highlight:: python
 
 API Documentation
 =================
 
-.. automodule:: borg.archiver
+.. automodule:: borg.logger
     :members:
     :undoc-members:
 
-.. automodule:: borg.archive
+.. automodule:: borg.upgrader
     :members:
     :undoc-members:
 
-.. automodule:: borg.repository
+.. automodule:: borg.archive
     :members:
     :undoc-members:
 
-.. automodule:: borg.remote
+.. automodule:: borg.cache
     :members:
     :undoc-members:
 
-.. automodule:: borg.cache
+.. automodule:: borg.key
     :members:
     :undoc-members:
 
-.. automodule:: borg.key
+.. automodule:: borg.lrucache
     :members:
     :undoc-members:
 
-.. automodule:: borg.keymanager
+.. automodule:: borg.archiver
     :members:
     :undoc-members:
 
-.. automodule:: borg.logger
+.. automodule:: borg.xattr
     :members:
     :undoc-members:
 
-.. automodule:: borg.helpers
+.. automodule:: borg.fuse
     :members:
     :undoc-members:
 
-.. automodule:: borg.locking
+.. automodule:: borg.keymanager
     :members:
     :undoc-members:
 
 .. automodule:: borg.shellpattern
     :members:
     :undoc-members:
 
-.. automodule:: borg.lrucache
+.. automodule:: borg.remote
     :members:
     :undoc-members:
 
-.. automodule:: borg.fuse
+.. automodule:: borg.helpers
     :members:
     :undoc-members:
 
-.. automodule:: borg.xattr
+.. automodule:: borg.locking
     :members:
     :undoc-members:
 
 .. automodule:: borg.platform
     :members:
     :undoc-members:
 
-.. automodule:: borg.platform_linux
+.. automodule:: borg.repository
     :members:
     :undoc-members:
 
 .. automodule:: borg.hashindex
     :members:
     :undoc-members:
 
-.. automodule:: borg.compress
-    :members: get_compressor, Compressor, CompressorBase
```

Comment:
this was lost.

the release todo checklist is in fact not quite complete here, but we have a bug for the build api being problematic at least.